subsys/nfc/lib/platform_internal_thread.c

```diff
@@ -24,8 +24,12 @@ LOG_MODULE_DECLARE(nfc_platform, CONFIG_NFC_PLATFORM_LOG_LEVEL);
 
 #define NFC_LIB_CTX_SIZE CONFIG_NFC_LIB_CTX_MAX_SIZE
 
+#define NFC_HDR_FLAG_NONE 0
+#define NFC_HDR_FLAG_COPY 1
+
 struct nfc_item_header {
-	uint16_t ctx_size;
+	uint8_t ctx_size;
```

Comment:
Shouldn't we add a range for `config NFC_LIB_CTX_MAX_SIZE`?

Reply:
Added